### PR TITLE
[RGPD] Add cookie banner and policy

### PR DESCRIPTION
**Pull request by JeremieSamson**
# Cookie banner

Capture du 2019-06-01 22-01-06

# Cookie policy

policy_1

policy_2

### CODE REVIEW

**Comment by victor-paumier** [comment]
Capture d’écran 2019-06-11 à 13 12 03
Are you really sure about the hover color?

File: templates/Home/cookiePolicy.html.twig [hunk 1]
```diff
@@ -12,51 +12,77 @@
         <div class="row mt-5">
             <h2 class="mb-4 text-secondary text-uppercase">{{ 'cookie_policy.description.title'|trans }}</h2>
             <div class="col-12">
-                {{ 'cookie_policy.description.content'|trans|raw|nl2br }}
+                <p>
+                    {{ 'cookie_policy.description.content1'|trans({'%url%': '<a target="_blank" rel="nofollow" href="https://buddysystem.eu">buddysystem.eu</a>'})|raw|nl2br }}
+                </p>
+                <p>
+                    {{ 'cookie_policy.description.content2'|trans({'%url%': '<a target="_blank" rel="nofollow" href="https://buddysystem.eu/fr/privacy">'~'footer.privacy'|trans({}, 'messages', app.user ? app.user.localeLanguage : default_locale)~'</a>'})|raw|nl2br }}
```
Comment (victor-paumier) [inline templates/Home/cookiePolicy.html.twig]: path('cookiePolicy')

File: features/home.feature [hunk 2]
```diff
@@ -40,3 +40,14 @@ Feature: Home
...
     And the response status code should be 200
+
+  @javascript
+  @wip
+  Scenario: As a visitor, I must see the cookie policy banner
+    Given I am on "/en"
+    Then the ".cc-message" element should be visible
+    And I should see "This website uses cookies to ensure you get the best experience. Learn more"
+    And I should see "Got it!"
+    When I click the ".cc-dismiss" element
```
Comment (victor-paumier) [inline features/home.feature]: `When I press "Got it!"` do the trick

**Comment by victor-paumier** [comment]
> Capture d’écran 2019-06-11 à 13 12 03
> Are you really sure about the hover color?

**Comment by victor-paumier** [comment]
Cancel close PR

File: templates/Home/cookiePolicy.html.twig (hunk 1 above)
Comment (JeremieSamson) [reply templates/Home/cookiePolicy.html.twig]: path('privacy')

**Comment by JeremieSamson** [comment]
> > Capture d’écran 2019-06-11 à 13 12 03
> > Are you really sure about the hover color?
> 
> IMO we should have `#63032f` on hover (same as sign up button)

Didn't find any reference into `_colors.scss` file about this one

File: translations/messages.en.yml [hunk 3]
```diff
@@ -142,6 +143,28 @@ terms_of_use.applicable.title: '14. Applicable law – Dispute'
...
 terms_of_use.legal.title: '15. Legal notices'
 terms_of_use.legal.block: "For any question, you can contact ESN France - Buddy System administration using this contact form.\nVersion dated 11 June 2018\n<ul><li><a target='_blank' href='http://www.ixesn.fr'>About ESN France</a></li>\n<li><a href='%url2%'>Buddy System experience</a></li>\n<li><a href='%url3%'>Buddy System Erasmus+ project</a></li></ul>\nTerms & Privacy Policy\n© 2018 Buddy System\nBy using our website, you accept that we use cookies to perform analytics and produce content & ads tailored to your interests.\nRead our policy on cookies usage.\n"
+cookie_policy.title: 'Cookie Policy'
+cookie_policy.description.title: '1. Description'
+cookie_policy.description.content1: 'This cookie policy (&quot;Policy&quot;) describes what cookies are and how the Website Operator (&quot;Website Operator&quot;, &quot;we&quot;, &quot;us&quot; or &quot;our&quot;) uses them on the %url% website and any of its products or services (collectively, &quot;Website&quot; or &quot;Services&quot;).'
```
Comment (victor-paumier) [inline translations/messages.en.yml]: remove &quot;

**Comment by victor-paumier** [comment]
> > > Capture d’écran 2019-06-11 à 13 12 03
> > > Are you really sure about the hover color?
> > 
> > 
> > IMO we should have `#63032f` on hover (same as sign up button)
> 
> Didn't find any reference into `_colors.scss` file about this one

https://github.com/ESNFranceG33kTeam/sf_buddysystem/blob/2776349b861389f8a51d15217a207afb85ebd536/assets/sass/main.scss#L71

Could you create a variable in _color.scss and change both plz

**Pull request merged**